scrapers/employees/employees.ts

```diff
@@ -18,29 +18,45 @@ class NeuEmployee {
     this.people = [];
   }
 
-  parseParameter(parameter: string): string {
-    // The NEU API returns 'Not Avaiable' for some fields instead of making them null
-    return parameter.toLowerCase() !== "not available" ? parameter : null;
+  /**
+   * The NEU API returns 'Not Avaiable' for some fields instead of making them null.
+   * Given an optional string, return null if the string is undefined or 'Not Available'
+   */
+  private parseOptionalParameter(parameter?: string): string | null {
+    if (!parameter || parameter.toLocaleLowerCase() === "not available") {
+      return null;
+    }
+    return parameter;
+  }
+
+  /**
+   * Given the raw data for one employee from NEU's API, convert it to our
+   * expected Employee type.
+   */
+  private parseEmployee(employee: EmployeeRequestResponse): Employee {
+    const email = this.parseOptionalParameter(employee.Email);
+    const primaryRole = this.parseOptionalParameter(employee.PositionTitle);
+    const longPrimaryRole = this.parseOptionalParameter(
+      employee.LongPositionTitle
+    );
+
+    return {
+      id: email ? email : uuidv4(),
```

Comment:
for consistency with `primaryRole` (also, seems cleaner to null coalesce here)
```suggestion
      id: email ?? uuidv4(),
```